vars/mtaBuild.groovy

```diff
@@ -24,35 +45,33 @@ def call(Map parameters = [:]) {
 
         def mtarFileName = "${id}.mtar"
 
-        def mtaJar = getMtaJar(parameters)
+        def mtaJar = getMtaJar(stepName, configuration)
 
         sh  """#!/bin/bash
             export PATH=./node_modules/.bin:${PATH}
             java -jar ${mtaJar} --mtar ${mtarFileName} --build-target=${buildTarget} build
             """
 
         def mtarFilePath = "${pwd()}/${mtarFileName}"
-        script.commonPipelineEnvironment.setMtarFilePath(mtarFilePath)
+        script?.commonPipelineEnvironment?.setMtarFilePath(mtarFilePath)
 
         return mtarFilePath
     }
 }
 
-private getMtaJar(parameters) {
+private getMtaJar(stepName, configuration) {
+
     def mtaJarLocation = 'mta.jar' //default, maybe it is in current working directory
 
-    if(parameters?.mtaJarLocation){
-        mtaJarLocation = "${parameters.mtaJarLocation}/mta.jar"
-        echo "[mtaBuild] MTA JAR \"${mtaJarLocation}\" retrieved from parameters."
-        return mtaJarLocation
+    if (configuration?.mtaJarLocation) {
+        mtaJarLocation = "$configuration.mtaJarLocation/mta.jar"
+        echo "[$stepName] MTA JAR '$mtaJarLocation' retrieved from configuration."
```

Comment:
will this replace stepName -> shouldn't it be `echo "[${stepName}]`?

Reply:
https://github.com/SAP/jenkins-library/blob/master/CONTRIBUTING.md#do-not-use-curly-braces--for-variables-or-variableproperty

Reply:
I suggest to change this in the contributing.md or at least remove the phrase to allow both.

We use `"${variable}"` in most other places in this repo.
From my perspective this is much simpler to grasp.

@marcusholl, @CCFenner : what do you think?

Reply:
In my personal opinion ${foo} is better to read as $foo. But I'm able to read any code as long as it compiles ;-).